extensions/aria_extension_tosca/simple_v1_0/modeling/__init__.py

```diff
@@ -497,8 +497,8 @@ def get(name, default=None):
 
 
 def create_workflow_operation_template_model(context, service_template, policy):
-    model = OperationTemplate(name=policy._name,
-                              service_template=service_template)
+    model = OperationTemplate(name=policy._name)
+    service_template.workflow_templates[model.name] = model
```

Comment:
since we use backpopulates, these fields are populated upon commit, we get a weird (temporary) behavior where in previous code `service_template.workflows` is a dict which has `None` as key for the value of model.

This needs to be addressed throughout the code.